pkg/daemon/ceph/agent/flexvolume/manager/fake.go

```diff
@@ -48,3 +49,10 @@ func (f *FakeVolumeManager) Detach(image, pool, id, key, clusterName string, for
 	}
 	return nil
 }
+
+func (f *FakeVolumeManager) Resize(image, pool, clusterName string, size uint64) error {
```

Comment:
I'm struggling with terminology here. Resize suggests grow and shrink, but expand suggests only grow. If this only allows growing, what happens if this gets negative valaues?